client/me/help/help-contact/index.jsx

```diff
@@ -204,6 +215,24 @@ module.exports = React.createClass( {
 
 	onOperatorsAway: function() {
 		this.showOperatorAvailabilityNotice( false );
+
+		//Autofill the subject field since we will be showing it now that operators have went away.
+		this.autofillSubject();
+	},
+
+	/**
+	 * Auto fill the subject with the first five words contained in the message field of the contact form.
+	 */
+	autofillSubject: function() {
+		if ( ! savedContactForm.message || savedContactForm.subject ) {
+			return;
+		}
+
+		const words = savedContactForm.message.split( /\s+/ );
+
+		savedContactForm.subject = words.slice( 0, 5 ).join( ' ' ) + '…';
+
+		this.forceUpdate();
```

Comment:
@dmsnell, I moved `savedContactForm` out of `state` because it was causing a re-render of the entire stack for every key stroke. I only needed a re-render in the case where the operators have went away, theres text in the message body, and the subject field is empty. That case for the most part will happen exactly once while a user is on the page if it happens at all.